Installation/CHANGES.md

```diff
@@ -23,11 +30,17 @@ Release date: September 2018
 -   Added a read-write property map to convert on-the-fly geometric
     objects from Cartesian kernels.
 
-### 2D Triangulations
+### 2D and 3D Triangulations
 
 -   Added a new type of intersection to handle the insertion of intersecting constraints
     in a `Constrained_triangulation_2`.
 
+-   The long-deprecated class `Triangulation_cell_base_with_circumcenter_3` and its associated
+    concept have been removed. Users should use the classes `Delaunay_cell_base_with_circumcenter_3`
+    or `Regular_cell_base_with_circumcenter_3`, depending on which type of triangulation they are using.
+-   The deprecated functions `mirror_index` and `mirror_vertex` of the class `Triangulation_face_base_2`
+    have been removed. Users should use the equivalent functions from the class `Triangulation_2`.
```

Comment:
I think you should add  `**Breaking change**: ` for those two items, because that removes deprecated class and functions.